module/web/ServerThread.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, pycore):
         self.cert = pycore.config["ssl"]["cert"]
         self.key = pycore.config["ssl"]["key"]
         self.host = pycore.config['webinterface']['host']
+        if self.host == "0.0.0.0" and pycore.config['webinterface']['dualstack']:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible binding to all interfaces.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5459581)